flink-contrib/docker-flink/base/Dockerfile

```diff
@@ -38,12 +38,12 @@ ENV JAVA_HOME /usr/java/default/
 RUN echo 'root:secret' | chpasswd
 
 #SSH as root... probably needs to be revised for security!
-RUN sed -i 's/PermitRootLogin without-password/PermitRootLogin yes/' /etc/ssh/sshd_config
```

Comment:
Agreed. It might break previous ubuntu versions workflow.
Changing the sed expression to the following works for precise/trusty/xenial

`sed -i 's/PermitRootLogin .*/PermitRootLogin yes/' /etc/ssh/sshd_config`

Also, currently the flink version is hardcoded in a URL in flink/Dockerfile which can be moved to **install-flink.sh** where flink/scala/hadoop versions can be specified. Something like this:

```
BASE_URL=http://www-us.apache.org/dist/flink/
FLINK_VERSION=1.0.3
SCALA_VERSION=2.11
HADOOP_VERSION=27 #2.7.0
FULL_URL="$BASE_URL/flink-$FLINK_VERSION/flink-$FLINK_VERSION-bin-hadoop$HADOOP_VERSION-scala_$SCALA_VERSION.tgz"

wget -q -O - $FULL_URL | tar -zxvf - -C /usr/local/

cd /usr/local && ln -s "./flink-$FLINK_VERSION" flink

```